packages/tests/tests/filter/subscribe.node.spec.ts

```diff
@@ -51,6 +51,9 @@ describe("Waku Filter V2: Subscribe", function () {
   });
 
   it("Subscribe and receive messages via lightPush", async function () {
+    // Nwaku subscribe to the default pubsub topic
+    await nwaku.sendSubscriptions();
```

Comment:
What if this is done but `nwaku` subscription is not tested - it shouldn't affect anything, correct?

Reply:
:thinking: I'm hesitant because the behavior of Nwaku will be a bit more complex regarding subscriptions.

But i don't see how that would impact tests and it's simpler.